src/index.tsx

```diff
@@ -7,6 +7,17 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import 'bootstrap/dist/js/bootstrap.bundle.min.js';
 import './styles/main.scss';
 
+let runtimeURL = process.env.MARLOWE_RUNTIME_WEB_URL;
+await fetch('/config.json').then(async(res) => {
+  if (res.status === 200) {
+    const {marloweWebServerUrl} = await res.json();
+    if (!!marloweWebServerUrl) {
+      runtimeURL = marloweWebServerUrl;
+    }
+  }
+});
+
+const hasValidRuntimeInstance = runtimeURL !== undefined && runtimeURL !== null && runtimeURL !== '' && runtimeURL.startsWith('http');
```

Comment:
there is a healthcheck endpoint available in the sdk, you should call it instead

Reply:
This check is not good enough

Reply:
It's something along these lines if I'm not mistaken:

```ts
import { mkRestClient } from "@marlowe.io/runtime-rest-client";

const restClient = mkRestClient(runtimeURL)

const hasValidRuntimeInstance = await restClient.healthcheck()
```

Reply:
I updated this.